lib/scrabble.rb

```diff
@@ -1,7 +1,16 @@
+require 'pry'
 class Scrabble
 
   def score(word)
-    1
+    total = 0
+    if word == nil
+      total = nil
+    else
+      word.chars.each do |letter|
+        total += point_values[letter.upcase]
+      end
+    end
+    total
```

Comment:
Seems pretty straightforward. Would consider breaking this method into smaller methods in order to make each step particularly apparent in this method. Has the added benefit of allowing you to create unit tests.